YDS-iOS/Source/Foundation/YDSIcon.swift

```diff
@@ -98,5 +98,13 @@ extension UIImage {
         }
         return image
     }
+    
+    internal func resize(to: CGFloat) -> UIImage {
```

Comment:
```suggestion
    internal func resize(to length: CGFloat) -> UIImage {
```
호출할 때는 to지만, 내부에선  length 같은게 가독성이 더 좋을 듯 합니다

Reply:
오 좋은 아이디어인거 같아요 수정해두겠습니다

Reply:
수정했습니다~